tensorboard/plugins/image/tf_image_dashboard/tf-image-loader.html

```diff
@@ -34,34 +34,47 @@
 -->
 <dom-module id="tf-image-loader">
   <template>
-    <tf-card-heading tag="[[tag]]" run="[[run]]" color="[[_runColor]]">
+    <tf-card-heading
+      tag="[[tag]]"
+      run="[[run]]"
+      display-name="[[tagMetadata.displayName]]"
+      description="[[tagMetadata.description]]"
+      color="[[_runColor]]"
+    >
+      <template is="dom-if" if="[[_hasMultipleSamples]]">
+        <div>sample: [[_sampleText]] of [[ofSamples]]</div>
+      </template>
       <template is="dom-if" if="[[_hasAtLeastOneStep]]">
-        step
-        <span style="font-weight: bold">[[_stepValue]]</span>
-        <template is="dom-if" if="[[_currentWallTime]]">
-          ([[_currentWallTime]])
-        </template>
-        <paper-spinner-lite active hidden$=[[!_isImageLoading]]>
-        </paper-spinner-lite>
+        <div>
+          step
+          <span style="font-weight: bold">[[_stepValue]]</span>
+          <template is="dom-if" if="[[_currentWallTime]]">
+            ([[_currentWallTime]])
+          </template>
+          <paper-spinner-lite active hidden$=[[!_isImageLoading]]>
+          </paper-spinner-lite>
+        </div>
       </template>
       <template is="dom-if" if="[[_hasMultipleSteps]]">
-        <paper-slider
-          id="steps"
-          immediate-value="{{_stepIndex}}"
-          max="[[_maxStepIndex]]"
-          max-markers="[[_maxStepIndex]]"
-          snaps
-          step="1"
-          value="{{_stepIndex}}"
-        ></paper-slider>
+        <div>
+          <paper-slider
+            id="steps"
+            immediate-value="{{_stepIndex}}"
+            max="[[_maxStepIndex]]"
+            max-markers="[[_maxStepIndex]]"
+            snaps
+            step="1"
+            value="{{_stepIndex}}"
+          ></paper-slider>
+        </div>
       </template>
     </tf-card-heading>
     <div id="main-image-container"></div>
 
     <style>
       :host {
         display: block;
-        width: 330px;
+        width: 350px;
```

Comment:
+1 for the UI fix! I see what you mean - the spinner drops down to the second line.